assets/js/helper-options.js

```diff
@@ -50,4 +50,23 @@ jQuery(document).ready(function($) {
     if (localStorage.getItem('simple-tags-dismiss-rating')) {
       $('.simple-tags-review-box').hide();
     }
-});
\ No newline at end of file
+});
+
+// -------------------------------------------------------------
+//   Prevent negative number of input type number
+// -------------------------------------------------------------
+jQuery(document).on('change paste keyup keydown', 'input[type="number"]', function (e) {
```

Comment:
Hi @anhredweb, this is one of the reason i said "adding another option to accept min and max for the text input (**MOST PREFERED/APPROPRIATE IN THIS CASE**)" since we'll me modifying our sole source code and not site global codes.

For example, TaxoPress is been used by thousands of sites with different plugins and themes and varieties of features and there are definitely part of the site that uses number input field which by convention allow empty value or negative value like -1 for example. So, it's wrong for us to target all number input on the site which is what you implemented by using the selection on **input[type="number"]** which shows this won't be applicable to TaxoPress alone and we may be causing issues for other plugins or themes or even other part of TaxoPress integration that may accept - in/empty value for non required number field.